mint.json

```diff
@@ -295,12 +295,12 @@
     {
       "group": "Community & Support",
       "pages": [
-        "community-support/introduction",
-        "community-support/changelog",
-        "community-support/roadmap",
-        "community-support/machine-setup",
-        "community-support/run-in-local-machine",
-        "community-support/add-a-new-provider"
+        "community/introduction",
+        "community/changelog",
+        "community/roadmap",
+        "community/run-in-local-machine",
+        "community/machine-setup",
```

Comment:
Any reason why run in local machine is before machine set up? @jainpawan21

Reply:
Please can you change it back? because I read it and its good for machine setup to come before run-in-local-machine